components/measurement/FeedbackBox.js

```diff
@@ -22,29 +22,32 @@ const StyledError = styled.small`
   color: ${props => props.theme.colors.red5};
 `
 
-const okValues = ['ok', 'ok.unreachable', 'ok.broken', 'ok.parked']
 const blockedValues = [
   { top: 'ok' },
-  { top: 'ok.broken' },
-  { top: 'ok.parked' },
-  { top: 'ok.unreachable' },
-  { top: 'blocked',
-    sub: [
-  { 
-    top: 'blocked.blockpage',
+  { top: 'broken',
```

Comment:
Following the discussion on slack, let's for the moment limit this to only 2 options:
* down.unreachable
* down.misconfigured

The goal is to capture under the `down` tree anything that has to do with issues with the service itself that are not attributable to censorship.

`unreachable` is meant to capture anything that renders the resource unreachable from anywhere in the world. It's probably useful to capture this separately, because in this instance, when the blocking is implemented in a way that is indistinguishable from the service being unreachable (ex. timeout), the outcome is ambiguous. There are still several cases in which we can say that blocking happening, even if the service is unreachable (ex. DNS blocks, HTTP proxies, RST injection, TLS MITMs).

`misconfigured` is any form of misconfiguration that renders the resource still available, yet it's not in working state. This can include instances where the domain is parked, it's serving a blank page or in any case not functioning as expected, while still being reachable.